importer/trickle/trickle_test.go

```diff
@@ -443,7 +443,8 @@ func TestAppend(t *testing.T) {
 	r := bytes.NewReader(should[nbytes/2:])
 	blks, errs := chunk.Chan(chunk.NewSizeSplitter(r, 500))
 
-	nnode, err := TrickleAppend(nd, dbp.New(blks, errs))
+	ctx := context.TODO()
```

Comment:
maybe test contexts should be `context.Background()` so we can grep for `context.TODO()`?